### PR TITLE
Fix the List* APIs to only take experiment name.

### PR DESCRIPTION
**Pull request by dinogun**
This will remove the usage of `application_name` and instead only use `experiment_name`.

	Signed-off-by: <dgunigun@redhat.com>

### CODE REVIEW

**Comment by dinogun** [comment]
@chandrams This is ready for review now ... I will make the searchSpace changes in a separate PR.

File: src/main/java/com/autotune/analyzer/services/ListAppLayers.java [hunk 1]
```diff
@@ -83,59 +96,74 @@
...
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
         JSONArray outputJsonArray = new JSONArray();
-        resp.setContentType("application/json");
-
-        String applicationName = req.getParameter(AnalyzerConstants.ServiceConstants.APPLICATION_NAME);
+        resp.setContentType(JSON_CONTENT_TYPE);
 
-        for (String autotuneObjectKey : AutotuneDeployment.applicationServiceStackMap.keySet()) {
-            AutotuneObject autotuneObject = AutotuneDeployment.autotuneObjectMap.get(autotuneObjectKey);
+        /* Check if there are any experiments running at all ? */
```
Comment (chandrams) [inline src/main/java/com/autotune/analyzer/services/ListAppLayers.java:101]: Change it to single line comment

File: src/main/java/com/autotune/analyzer/services/ListAppLayers.java [hunk 2]
```diff
@@ -83,59 +96,74 @@
...
-            //No application_name parameter was passed in the request
-            if (applicationName == null) {
-                for (String application : AutotuneDeployment.applicationServiceStackMap.get(autotuneObjectKey).keySet()) {
-                    addLayerToResponse(outputJsonArray, autotuneObjectKey, autotuneObject, application);
-                }
-            } else {
-                addLayerToResponse(outputJsonArray, autotuneObjectKey, autotuneObject, applicationName);
+        String experimentName = req.getParameter(AnalyzerConstants.ServiceConstants.EXPERIMENT_NAME);
+        String layerName = req.getParameter(AnalyzerConstants.AutotuneConfigConstants.LAYER_NAME);
+        /* If experiment name is not null, try to find it in the hashmap */
```
Comment (chandrams) [inline src/main/java/com/autotune/analyzer/services/ListAppLayers.java:110]: Change all comments to single line comment

**Review by chandrams — APPROVED** [review]
LGTM

**Pull request merged**